.travis.yml

```diff
@@ -11,11 +11,15 @@ matrix:
       env: CGO_ENABLED=0
     - go: "1.12.x"
       env: CGO_ENABLED=0
+      env: GO111MODULE=on
     - go: "1.11.x"
       env: CGO_ENABLED=0
+      env: GO111MODULE=on
     - go: "1.13.x"
     - go: "1.12.x"
+      env: GO111MODULE=on
     - go: "1.11.x"
+      env: GO111MODULE=on
 
```

Comment:
As the `go.mod` file specifies that the mini go version is go1.13 we should just use that. 

Let's remove these